### DISCUSSION TITLE
What does "per-realm storage" mean

### DISCUSSION THREAD

**Discussion by zyairzy** [keycloak/keycloak#14813] [Q&A] [answered]
Hello, I'm learning about keycloak.X new storage.

In this blog https://www.keycloak.org/2022/07/storage-map.html, there is a sentence: Keycloak 19 brings in an early preview of the future Keycloak store supporting no-downtime upgrades, per-realm storage, and cloud-nativity from its very inception.

What does "per-realm storage" mean in above sentence? Anyone can help to explain it? Thank U

**Comment by sschu** [comment, marked as answer]
It means you can configure different storages for different realms.

**Reply by zyairzy** [reply]
> It means you can configure different storages for different realms.

Thanks. I also understand it literally like you said, but I can't find any relevant examples. Could you show some example or any other detail? Thank you

**Reply by sschu** [reply]
The implementation for that is not finished, yet.
